src/occupational_classification/_config/sic_soc_llm_config.toml

```diff
@@ -0,0 +1,20 @@
+[sic_soc_data]
+
+[lookups]
+sic_structure = "notebooks/publisheduksicsummaryofstructureworksheet.xlsx"
+# https://www.ons.gov.uk/file?uri=/methodology/classificationsandstandards/ukstandardindustrialclassificationofeconomicactivities/uksic2007/publisheduksicsummaryofstructureworksheet.xlsx
+sic_index = "notebooks/uksic2007indexeswithaddendumdecember2022newformatsept20241.xlsx"
+# https://www.ons.gov.uk/file?uri=/methodology/classificationsandstandards/ukstandardindustrialclassificationofeconomicactivities/uksic2007/uksic2007indexeswithaddendumdecember2022newformatsept20241.xlsx
+
+soc_structure = "notebooks/data/soc_data/soc2020volume1structureanddescriptionofunitgroupsexcel16102024.xlsx"
+
+soc_index = "notebooks/data/soc_data/soc2020volume2thecodingindexexcel16102024.xlsx"
+
+sic_condensed = "sic_2d_condensed.txt"
```

Comment:
Same as above, only keep what you'll need for this repo/soc_hierarchy functionality